### PR TITLE
handle empty repo

### PR DESCRIPTION
**Pull request by eliwind**
... and also blacken the code

### CODE REVIEW

**Comment by eliwind** [comment]
CircleCI is not properly configured for this repo -- disregard the failing check

File: jf_agent/bb_download.py [hunk 1]
```diff
@@ -88,23 +90,22 @@ def get_default_branch_commits(client, repos):
...
-            commits = api_project.repos[repo['name']].commits(
-                until=repo['default_branch_name'])
+            commits = api_project.repos[repo['name']].commits(until=repo['default_branch_name'])
+
+            for commit in tqdm(
+                commits,
+                desc=f'downloading {repo["project"]["login"]}/{repo["name"]} commits',
+                unit='commit',
+            ):
+                yield _normalize_commit(commit, repo)
```
Comment (ametzger) [inline jf_agent/bb_download.py]: @eliwind to confirm, this is the only non-formatting change?

Comment (eliwind) [reply jf_agent/bb_download.py]: That's right

Comment (eliwind) [reply jf_agent/bb_download.py]: the problem was that we had a try/catch around the thing that returns a generator of commits, but the NotFound wasn't actually thrown until you iterator over them

**Pull request merged**